easybuild/easyblocks/g/gromacs.py

```diff
@@ -102,6 +104,12 @@ def get_gromacs_arch(self):
             # According to [2] the performance difference between SSE2 and SSE4.1 is minor on x86
             # and SSE4.1 is not supported by AMD Magny-Cours[1].
             res = 'SSE2'
+        elif optarch == OPTARCH_GENERIC:
+            cpu_arch = get_cpu_architecture()
+            if cpu_arch == X86_64:
+                res = 'SSE2'
+            else:
+                res = 'None'
```

Comment:
@victorusu Is `-DGMX_SIMD=None` a correct setting? Maybe we should include a reference to the documentation for this in a comment above this line?

Reply:
@boegel, this is a link to the 'None' option http://manual.gromacs.org/documentation/2018/install-guide/index.html#simd-support

Reply:
@victorusu Please include that link as a comment *in* the easyblock, for future reference